cpp/ps0000/ps0000_soln.cpp

```diff
@@ -1,7 +1,8 @@
-#include <iostream>
+#include <iostream> // 2/2
 using namespace std;
-int main()
+int main() // 1/1
 {
-    cout << "hello world" << endl;
-    return 0;
+    cout << "Hello, World!" << endl; // 3/4
+    return 0; // 1/1
 }
+// great work 😎 9/10 ⭐

```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit